teorija.tex

```diff
@@ -680,7 +678,7 @@ \subsection{Ponazoritev realnih in racionalnih števil}
 
 \begin{figure}[ht]
   \begin{center}
-    \includegraphics[width=0.8\textwidth]{slike/realrac.pdf}
+    %\includegraphics[width=0.8\textwidth]{slike/realrac.pdf}
```

Comment:
Tole mi je pa zgleda kaksna napaka pri merganju ker sm to vrstico mel ze odkomentirano.